providers/implementations/signature/dsa_sig.c

```diff
@@ -26,27 +26,32 @@
 #include "internal/nelem.h"
 #include "internal/sizes.h"
 #include "internal/cryptlib.h"
+#include "crypto/dsa.h"
 #include "prov/providercommon.h"
 #include "prov/implementations.h"
 #include "prov/provider_ctx.h"
 #include "prov/securitycheck.h"
 #include "prov/fipsindicator.h"
-#include "crypto/dsa.h"
 #include "prov/der_dsa.h"
 
 static OSSL_FUNC_signature_newctx_fn dsa_newctx;
 static OSSL_FUNC_signature_sign_init_fn dsa_sign_init;
 static OSSL_FUNC_signature_verify_init_fn dsa_verify_init;
 static OSSL_FUNC_signature_sign_fn dsa_sign;
+static OSSL_FUNC_signature_sign_message_update_fn dsa_signverify_message_update;
+static OSSL_FUNC_signature_sign_message_final_fn dsa_sign_message_final;
 static OSSL_FUNC_signature_verify_fn dsa_verify;
+static OSSL_FUNC_signature_verify_message_update_fn dsa_signverify_message_update;
+static OSSL_FUNC_signature_verify_message_final_fn dsa_verify_message_final;
 static OSSL_FUNC_signature_digest_sign_init_fn dsa_digest_sign_init;
-static OSSL_FUNC_signature_digest_sign_update_fn dsa_digest_signverify_update;
+static OSSL_FUNC_signature_digest_sign_update_fn dsa_digest_sign_update;
```

Comment:
Not sure why this name was changed (it does both sign and verify)

Reply:
@levitte -  this does not seem to be resolved

Reply:
Okie